test_runner/src/main/kotlin/ftl/util/FlankException.kt

```diff
@@ -69,3 +69,12 @@ class FlankCommonException(message: String) : FlankException(message)
  *  @param message [String] message to be printed to [System.err]
  */
 class FlankConfigurationException(message: String) : FlankException(message)
+
+/**
+ *  Exception throws when required yaml file not found
+ *
+ *  Exit code: 1
+ *
+ *  @param message [String] message to be printed to [System.err]
+ */
+class FlankNoSuchFileException(message: String) : FlankException(message)
```

Comment:
I am not sure about creating additional exception that would handled the same way as other two. Intention behind `FlankCommonException` was to wrap all errors caused by user. But this is just my opinion. :) 
If you decide to refactor this, please take care of `FlankConfiguratinnException`

Reply:
On the second thought I think we should use `FlankFatalError` instead. It is used to indicate users mistakes. Also, exit code will be inline with documentation https://github.com/Flank/flank#exit-codes

Reply:
I think you have right. I change this exceptions to ` FlankFatalError ` with return code: 2. 
Thanks for review! :)